app/views/shared/_application_information.html.erb

```diff
@@ -43,6 +43,14 @@
             <td class="govuk-table__cell"><strong>Location:</strong></td>
             <td class="govuk-table__cell"><%= link_to 'View site on Google Maps', map_link(@planning_application.full_address), target: '_blank', class: "govuk-link" %></td>
           </tr>
+          <tr class="govuk-table__row">
```

Comment:
we'd also need a little catch statement here for all the previous planning applications that were creted without ward.